juriscraper/oral_args/united_states/state/miss.py

```diff
@@ -0,0 +1,132 @@
+"""Scraper for [Full name of court]
+CourtID: [unique abbreviation to be used by software/filesystem]
+Court Short Name: [standard abbreviation used in citations]
+Author:
+Reviewer:
+History:
+  YYYY-MM-DD: Created by XXX
+"""
+import os
+import calendar
+
+from lxml import html
+from juriscraper.OralArgumentSite import OralArgumentSite
+from juriscraper.lib.string_utils import titlecase
+from juriscraper.lib.string_utils import convert_date_string
+
+DAYS = [d.lower() for d in calendar.day_name]
+
+def get_first_word(text):
+    return text.split()[0]
+
+
+def make_year_iterable(year):
+    "returns a (year, sitting) tuple"
+    iterable = list()
+    for sitting in range(1, 7):
+        iterable.append((year, sitting))
+    return iterable
+
+
+def make_back_scrape_iterable():
+    iterable = [(2014, 5), (2014, 6)]
+    for year in range(2015, 2019):
+        iterable += make_year_iterable(year)
+    return iterable
+
+
+def parse_sibling_text_nodes(iframe):
+    textlist = iframe.xpath("./following-sibling::text()")
+    textlist = [el.strip() for el in textlist
+                if el.strip() and el.strip() != "\\r\\n"]
```

Comment:
It looks like I initially thought this as well, but it seems that ```.strip()``` didn't work that way.

Reply:
Hm, [the docs](https://docs.python.org/2/library/string.html#string.strip) say that strip will remove "whitespace". I'd expect that to include `\r` and `\n`, right?

Reply:
I agree wholeheartedly with you, and also expected this myself.  Perhaps the "\\" prevents the strip method from recognizing this as a line ending.  My original code followed your expectation, but didn't work, which is why I had to add the extra code.  I also agree that it is ugly, but I'm not sure we can do better, since we depend on how the webmasters create the page.  Your comment on this provoked me to clean it up a bit and place the filter in the list constructor, which makes the code a bit cleaner, but really doesn't do what we both seem to want.

Also, this code was necessary for the one place I found where the case date is not within a "b" or "strong" element.  All other pages seem to follow the practice of placing the date in a "bold" element.  This looks to be the very first archive page created by the IT staff at the court, so it may be an anomaly that will not occur again.  Here is the url, so that you can test the strip method on your side.

https://courts.ms.gov/appellatecourts/sc/archive/2014/scs52014.php

Reply:
OK. I'll chalk this up to: "Scrapers are gross and sometimes have gross things."